src/rdkafka.h

```diff
@@ -3753,6 +3761,8 @@ void rd_kafka_set_logger(rd_kafka_t *rk,
  *
  * If the \p \"debug\" configuration property is set the level is automatically
  * adjusted to \c LOG_DEBUG (7).
+ *
+ * @deprecated Set the \c "log_level" configuration property instead.
```

Comment:
Should the debug configuration in Configuration.md as well?

Reply:
Should debug as well what? Be deprecated? No, it is very much needed still (still a few more bugs in the code somewhere!)

Reply:
I misinterpreted what was being said. Carry on